guides/internals.rst

```diff
@@ -207,34 +207,27 @@ The Shrinker
 ------------
 
 The shrinking part of Hypothesis is organised into a single class called ``Shrinker``
-that lives in ``engine.py``.
+that lives in ``hypothesis/internal/conjecture/shrinker.py``.
 
 Its job is to take an initial ``ConjectureData`` object and some predicate that
 it satisfies, and to try to produce a simpler ``ConjectureData`` object that
 also satisfies that predicate.
 
-~~~~~~~~~~~~~~
-Search Process
-~~~~~~~~~~~~~~
+The search process mostly happens in the ``shrink`` method, which tries various
+shrink passes in the ``greedy_shrink`` method and then reports on the outcome.
+In previous versions, we followed this by calling an ``escape_local_minimum``
```

Comment:
The rest of this paragraph doesn't seem to be very useful, even as historical context.

I'd be inclined to stop after mentioning `greedy_shrink`, especially in light of the next paragraph that (rightly) recommends consulting the source directly.